hack/queue_conf.py

```diff
@@ -47,10 +49,23 @@ def set_queue_attributes(queue_names):
                 'MessageRetentionPeriod': str(int(timedelta(days=14).total_seconds()))
             }
         )
-
         _logger.debug("Remote responded with %r after setting queue attributes for %r",
                       response, queue_name)
 
+        if _DEPLOYMENT_PREFIX:
+            response = client.tag_queue(
+                QueueUrl=queue_url,
+                Tags={
+                    'ENV': _DEPLOYMENT_PREFIX
+                }
+            )
+            _logger.debug("Remote responded with %r after setting queue tag for %r",
+                          response, queue_name)
+            # Tagging API actions are limited to 5 TPS (Transactions Per Second) per AWS account.
+            # http://docs.aws.amazon.com/AWSSimpleQueueService/latest/
+            # SQSDeveloperGuide/limits-queues.html
+            sleep(1 / 5)
```

Comment:
[This](https://github.com/fabric8-analytics/fabric8-analytics-worker/pull/468/files#diff-8c255b087ddc69d83b81d4fb3a18c9cbR26) will run once per deployment, but we have `-ingestion`, `-api`, ... deployments - it's a mess. And they will all execute this script +/- at the same time.

Reply:
They won't since #468

Reply:
Check latest commit please.